runtime/libs/nnapi/include/NeuralNetworksTypes.h

```diff
@@ -172,6 +176,7 @@ typedef int (*ANeuralNetworksExecution_setMeasureTiming_fn)(ANeuralNetworksExecu
 typedef int (*ANeuralNetworksExecution_getDuration_fn)(const ANeuralNetworksExecution *execution,
                                                        int32_t durationCode, uint64_t *duration);
 
+#if __ANDROID_API__ >= 30
```

Comment:
The same as I commented in `NeuralNetworksShim.h`. First four functions below this macro seems to be supported at API level 29.